Assets/Scripts/Utilities/SelfDestruct.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+/// <summary>
+/// Helper MonoBehaviour to destroy a GameObject after a set timer.
+/// </summary>
+public class SelfDestruct : MonoBehaviour
+{
+    [SerializeField] private float time;
+
+    private float aliveFor;
+
+    // Update is called once per frame
+    void Update()
+    {
+        aliveFor += Time.deltaTime;
+        if (time <= aliveFor)
+        {
+            Destroy(gameObject);
+        }
```

Comment:
This could just be `Destroy(time);` in `Start()`. 
Or use a coroutine if you need more control